lib/kitchen/driver/dokken.rb

```diff
@@ -444,6 +448,22 @@ def repo(image)
         parse_image_name(image)[0]
       end
 
+      def tag(image)
```

Comment:
You wanna toss some yard on these. I know we don't have anything elsewhere, but it's a good time to start

Reply:
done!